packages/stark-ui/base.spec.ts

```diff
@@ -18,3 +18,6 @@ import { TestBed } from "@angular/core/testing";
 import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from "@angular/platform-browser-dynamic/testing";
 
 TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+
+// define global environment variable (used in some places in stark-ui)
+global["ENV"] = "development";
```

Comment:
Should not this variable be defined also in base.spec.ts file of Starter and Showcase projects ?
Otherwise, I've the feeling that the result won't be the one that we want...

Reply:
Indeed, I missed those, I'll add them ;)